### ISSUE TITLE
Verify 2.0 with conda on windows

### ISSUE THREAD

**Issue by daf** [ioos/compliance-checker#189]
@abirger [reports](https://groups.google.com/d/msg/ioos_tech/q39_0Z0iUlo/a88ZVwN-BQAJ) that updating 2.0 using pip in an anaconda environment on windows isn't working for him. Investigate.

**Comment by ocefpaf** [comment]
Anaconda or no `pip install` won't work  on Windows unless we create wheels for compliance-checker.

PS: I did just fixed the Windows build on the IOOS channel and the [new binaries are up](http://anaconda.org/IOOS/compliance-checker/files).

**Comment by abirger** [comment]
@ocefpaf , @daf , I've just verified that compliance-checker update with `conda update compliance-checker` command goes well, and the updated compliance-checker works OK (at least with the test files).

Need to notice though, that I had to re-install Anaconda from scratch using 32-bit instead of 64-bit because the 64-bit Anaconda stopped activating the compliance_checker environment after upgrade to the last version (even with the compliance-checker v1.1.1). It was not because of the path, I am sure that something happened with the access rights to the default Anaconda2 installation place but it was much easier to remove 64-bit version and replace it with the 32-bit one than to fix whatever 64-bit version got screwed up.
